components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/idp/sso/configurator/IS500SAMLSSOConfigurator.java

```diff
@@ -115,12 +115,21 @@ public boolean createProvider(SSOProvider provider) {
 
     @Override
     public boolean createProvider(WebApp webApp) {
+        String acsUrl = webApp.getAcsURL().trim();
         SSOProvider ssoProvider = webApp.getSsoProviderDetails();
         if (ssoProvider == null) {
             log.warn("No SSO Configurator details given. Manual setup of SSO Provider required.");
         }
 
-        ssoProvider.setAssertionConsumerURL(SSOConfiguratorUtil.getGatewayUrl(webApp));
+        if(acsUrl != null) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/f3131fd198c08449ff724321a95ba67682999cc3